### PR TITLE
27 adding the navigation tree of the text viewer sections in the panel

### PR DESCRIPTION
**Pull request by cienco**
merge and review the #154 before this one

### CODE REVIEW

**Comment by MPapini91** [comment]
@cienco  , could you please use English for documentation? Thank you

**Comment by cienco** [comment]
> @cienco , could you please use English for documentation? Thank you

sure it's done

File: src/app/pages/workspace/workspace-text-window/workspace-text-window.component.ts
Comment (MPapini91) [inline src/app/pages/workspace/workspace-text-window/workspace-text-window.component.ts:1]: What was the problem with Firefox in this case?

Comment (cienco) [reply src/app/pages/workspace/workspace-text-window/workspace-text-window.component.ts:1]: scrolling down was not working in firefox only

File: src/app/services/workspace.service.ts [hunk 1]
```diff
@@ -385,4 +386,8 @@ export class WorkspaceService {
   public retrieveResourceElementById(resourceId: number): Observable<ResourceElement> {
     return this.http.get<ResourceElement>(`${this.textoUrl}/resource/${resourceId}`);
   }
+
```
Comment (MPapini91) [inline src/app/services/workspace.service.ts:389]: Add jsdoc

**Review by MPapini91 — CHANGES_REQUESTED** [review]

**Review by MPapini91 — APPROVED** [review]

**Pull request merged**